### PR TITLE
Add minimum requirement in arm when deploying TiFlash

### PR DESCRIPTION
**Pull request by Lloyd-Pottiger**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Add the minimum requirement when deploying TiFlash in Linux ARM64 architecture.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by shichun-0415** [comment]
Please confirm whether we need to update these pages as well:
- [TiDB 软件和硬件环境建议配置](https://docs.pingcap.com/zh/tidb/stable/hardware-and-software-requirements#%E5%BC%80%E5%8F%91%E5%8F%8A%E6%B5%8B%E8%AF%95%E7%8E%AF%E5%A2%83)
   
   image

- [6.3 release notes](https://docs.pingcap.com/zh/tidb/stable/release-6.3.0#%E5%85%B6%E4%BB%96)
   
   image

- [使用 TiUP 升级 TiDB - 兼容性检查](https://docs.pingcap.com/zh/tidb/stable/upgrade-tidb-using-tiup#1-%E5%8D%87%E7%BA%A7%E5%85%BC%E5%AE%B9%E6%80%A7%E8%AF%B4%E6%98%8E)
  
  image

File: tiflash/tiflash-overview.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ TiFlash 提供列式存储，且拥有借助 ClickHouse 高效实现的协处理
 
 TiFlash 以低消耗不阻塞 TiKV 写入的方式，实时复制 TiKV 集群中的数据，并同时提供与 TiKV 一样的一致性读取，且可以保证读取到最新的数据。TiFlash 中的 Region 副本与 TiKV 中完全对应，且会跟随 TiKV 中的 Leader 副本同时进行分裂与合并。
 
-在 Linux AMD64 架构的硬件平台上部署 TiFlash 时，CPU 必须支持 AVX2 指令集。确保命令 `cat /proc/cpuinfo | grep avx2` 有输出。通过使用 AVX2 指令集，TiFlash 的向量化引擎能提供更好的性能。
+在 Linux AMD64 架构的硬件平台上部署 TiFlash 时，CPU 必须支持 AVX2 指令集。确保命令 `cat /proc/cpuinfo | grep avx2` 有输出。通过使用 AVX2 指令集，TiFlash 的向量化引擎能提供更好的性能。而在 ARM 架构的硬件平台上部署 TiFlash 时，CPU 必须支持 ARMv8 指令集。
```
Comment (solotzg) [inline tiflash/tiflash-overview.md:20]: Can you provide any shell scripts to confirm it ?

Comment (Lloyd-Pottiger) [reply tiflash/tiflash-overview.md]: add `cat /proc/cpuinfo | grep 'crc32\|asimd'`

Comment (Lloyd-Pottiger) [reply tiflash/tiflash-overview.md]: `cat /proc/cpuinfo | grep 'crc32' | grep 'asimd'`

File: hardware-and-software-requirements.md [hunk 2]
```diff
@@ -94,7 +94,7 @@ TiDB 支持部署和运行在 Intel x86-64 架构的 64 位通用硬件服务器
 > - TiKV 的 SSD 盘推荐使用 NVME 接口以保证读写更快。
 > - 如果仅验证功能，建议使用 [TiDB 数据库快速上手指南](/quick-start-with-tidb.md)进行单机功能测试。
 > - TiDB 对于磁盘的使用以存放日志为主，因此在测试环境中对于磁盘类型和容量并无特殊要求。
-> - 从 v6.3.0 开始，在 Linux AMD64 架构的硬件平台下部署 TiFlash 时，CPU 必须支持 AVX2 指令集。确保命令 `cat /proc/cpuinfo | grep avx2` 有输出。通过使用 AVX2 指令集，TiFlash 的向量化引擎能提供更好的性能。
+> - 从 v6.3.0 开始，在 Linux AMD64 架构的硬件平台下部署 TiFlash 时，CPU 必须支持 AVX2 指令集。确保命令 `cat /proc/cpuinfo | grep avx2` 有输出。通过使用 AVX2 指令集，TiFlash 的向量化引擎能提供更好的性能。而在 ARM 架构的硬件平台上部署 TiFlash 时，CPU 必须支持 ARMv8 指令集。确保命令 `cat /proc/cpuinfo | grep 'crc32' | grep 'asimd'` 有输出。
```
Comment (solotzg) [inline hardware-and-software-requirements.md:97]: Can we make it `Linux ARM64` ?

Comment (Lloyd-Pottiger) [reply hardware-and-software-requirements.md]: done

**Review by solotzg — APPROVED** [review]
lgtm

**Comment by Lloyd-Pottiger** [comment]
@shichun-0415 Those three files also updated. Please take a look again.

File: hardware-and-software-requirements.md [hunk 3]
```diff
@@ -94,7 +94,7 @@ TiDB 支持部署和运行在 Intel x86-64 架构的 64 位通用硬件服务器
 > - TiKV 的 SSD 盘推荐使用 NVME 接口以保证读写更快。
 > - 如果仅验证功能，建议使用 [TiDB 数据库快速上手指南](/quick-start-with-tidb.md)进行单机功能测试。
 > - TiDB 对于磁盘的使用以存放日志为主，因此在测试环境中对于磁盘类型和容量并无特殊要求。
-> - 从 v6.3.0 开始，在 Linux AMD64 架构的硬件平台下部署 TiFlash 时，CPU 必须支持 AVX2 指令集。确保命令 `cat /proc/cpuinfo | grep avx2` 有输出。通过使用 AVX2 指令集，TiFlash 的向量化引擎能提供更好的性能。
+> - 从 v6.3.0 开始，在 Linux AMD64 架构的硬件平台下部署 TiFlash 时，CPU 必须支持 AVX2 指令集。确保命令 `cat /proc/cpuinfo | grep avx2` 有输出。通过使用 AVX2 指令集，TiFlash 的向量化引擎能提供更好的性能。而在 Linux ARM64 架构的硬件平台上部署 TiFlash 时，CPU 必须支持 ARMv8 指令集。确保命令 `cat /proc/cpuinfo | grep 'crc32' | grep 'asimd'` 有输出。
```
Comment (shichun-0415) [inline hardware-and-software-requirements.md:97]:
```suggestion
> - 从 v6.3.0 开始，在 Linux AMD64 架构的硬件平台下部署 TiFlash 时，CPU 必须支持 AVX2 指令集。确保命令 `cat /proc/cpuinfo | grep avx2` 有输出。而在 Linux ARM64 架构的硬件平台上部署 TiFlash 时，CPU 必须支持 ARMv8 指令集。确保命令 `cat /proc/cpuinfo | grep 'crc32' | grep 'asimd'` 有输出。通过使用 AVX2 和 ARMv8 指令集，TiFlash 的向量化引擎能提供更好的性能。
```

File: releases/release-6.3.0.md [hunk 4]
```diff
@@ -254,7 +254,7 @@ TiDB 版本：6.3.0-DMR
 * 日志备份支持 GCS 和 Azure Blob Storage 作为备份存储。
 * 日志备份功能兼容分区交换 (Exchange Partition) DDL。
 * 不再支持通过 `ALTER TABLE ...SET TiFLASH MODE ...` 语法启用或禁用 [FastScan](/develop/dev-guide-use-fastscan.md) 功能。从 v6.2.0 版本升级到 v6.3.0 版本时，在 v6.2.0 版本的 FastScan 设置将失效，但不影响数据的正常读取。你需要重新使用变量方式设置 FastScan。从 v6.2.0 及更早版本升级到 v6.3.0 时，所有会话默认不开启 FastScan 功能，而是保持一致性的数据扫描功能。
-* 在 Linux AMD64 架构的硬件平台下部署 TiFlash 时，CPU 必须支持 AVX2 指令集。确保命令 `cat /proc/cpuinfo | grep avx2` 有输出。通过使用 AVX2 指令集，TiFlash 的向量化引擎能提供更好的性能。
+* 在 Linux AMD64 架构的硬件平台下部署 TiFlash 时，CPU 必须支持 AVX2 指令集。确保命令 `cat /proc/cpuinfo | grep avx2` 有输出。通过使用 AVX2 指令集，TiFlash 的向量化引擎能提供更好的性能。而在 Linux ARM64 架构的硬件平台上部署 TiFlash 时，CPU 必须支持 ARMv8 指令集。确保命令 `cat /proc/cpuinfo | grep 'crc32' | grep 'asimd'` 有输出。
```
Comment (shichun-0415) [inline releases/release-6.3.0.md:257]:
```suggestion
* 在 Linux AMD64 架构的硬件平台下部署 TiFlash 时，CPU 必须支持 AVX2 指令集。确保命令 `cat /proc/cpuinfo | grep avx2` 有输出。而在 Linux ARM64 架构的硬件平台上部署 TiFlash 时，CPU 必须支持 ARMv8 指令集。确保命令 `cat /proc/cpuinfo | grep 'crc32' | grep 'asimd'` 有输出。通过使用 AVX2 和 ARMv8 指令集，TiFlash 的向量化引擎能提供更好的性能。
```

File: tiflash/tiflash-overview.md [hunk 5]
```diff
@@ -17,7 +17,7 @@ TiFlash 提供列式存储，且拥有借助 ClickHouse 高效实现的协处理
 
 TiFlash 以低消耗不阻塞 TiKV 写入的方式，实时复制 TiKV 集群中的数据，并同时提供与 TiKV 一样的一致性读取，且可以保证读取到最新的数据。TiFlash 中的 Region 副本与 TiKV 中完全对应，且会跟随 TiKV 中的 Leader 副本同时进行分裂与合并。
 
-在 Linux AMD64 架构的硬件平台上部署 TiFlash 时，CPU 必须支持 AVX2 指令集。确保命令 `cat /proc/cpuinfo | grep avx2` 有输出。通过使用 AVX2 指令集，TiFlash 的向量化引擎能提供更好的性能。
+在 Linux AMD64 架构的硬件平台上部署 TiFlash 时，CPU 必须支持 AVX2 指令集。确保命令 `cat /proc/cpuinfo | grep avx2` 有输出。通过使用 AVX2 指令集，TiFlash 的向量化引擎能提供更好的性能。而在 Linux ARM64 架构的硬件平台上部署 TiFlash 时，CPU 必须支持 ARMv8 指令集。确保命令 `cat /proc/cpuinfo | grep 'crc32' | grep 'asimd'` 有输出。
```
Comment (shichun-0415) [inline tiflash/tiflash-overview.md:20]:
```suggestion
在 Linux AMD64 架构的硬件平台上部署 TiFlash 时，CPU 必须支持 AVX2 指令集。确保命令 `cat /proc/cpuinfo | grep avx2` 有输出。而在 Linux ARM64 架构的硬件平台上部署 TiFlash 时，CPU 必须支持 ARMv8 指令集。确保命令 `cat /proc/cpuinfo | grep 'crc32' | grep 'asimd'` 有输出。通过使用 AVX2 和 ARMv8 指令集，TiFlash 的向量化引擎能提供更好的性能。
```

File: hardware-and-software-requirements.md [hunk 6]
```diff
@@ -94,7 +94,7 @@ TiDB 支持部署和运行在 Intel x86-64 架构的 64 位通用硬件服务器
 > - TiKV 的 SSD 盘推荐使用 NVME 接口以保证读写更快。
 > - 如果仅验证功能，建议使用 [TiDB 数据库快速上手指南](/quick-start-with-tidb.md)进行单机功能测试。
 > - TiDB 对于磁盘的使用以存放日志为主，因此在测试环境中对于磁盘类型和容量并无特殊要求。
-> - 从 v6.3.0 开始，在 Linux AMD64 架构的硬件平台下部署 TiFlash 时，CPU 必须支持 AVX2 指令集。确保命令 `cat /proc/cpuinfo | grep avx2` 有输出。通过使用 AVX2 指令集，TiFlash 的向量化引擎能提供更好的性能。
+> - 从 v6.3.0 开始，在 Linux AMD64 架构的硬件平台下部署 TiFlash 时，CPU 必须支持 AVX2 指令集。确保命令 `cat /proc/cpuinfo | grep avx2` 有输出。而在 Linux ARM64 架构的硬件平台上部署 TiFlash 时，CPU 必须支持 ARMv8 指令集。确保命令 `cat /proc/cpuinfo | grep 'crc32' | grep 'asimd'` 有输出。通过使用 AVX2 和 ARMv8 指令集，TiFlash 的向量化引擎能提供更好的性能。
```
Comment (shichun-0415) [inline hardware-and-software-requirements.md:97]:
```suggestion
> - 从 v6.3.0 开始，在 Linux AMD64 架构的硬件平台部署 TiFlash 时，CPU 必须支持 AVX2 指令集。确保命令 `cat /proc/cpuinfo | grep avx2` 有输出。而在 Linux ARM64 架构的硬件平台部署 TiFlash 时，CPU 必须支持 ARMv8 指令集。确保命令 `cat /proc/cpuinfo | grep 'crc32' | grep 'asimd'` 有输出。通过使用 AVX2 和 ARMv8 指令集，TiFlash 的向量化引擎能提供更好的性能。
```

File: tiflash/tiflash-overview.md [hunk 7]
```diff
@@ -17,7 +17,7 @@ TiFlash 提供列式存储，且拥有借助 ClickHouse 高效实现的协处理
 
 TiFlash 以低消耗不阻塞 TiKV 写入的方式，实时复制 TiKV 集群中的数据，并同时提供与 TiKV 一样的一致性读取，且可以保证读取到最新的数据。TiFlash 中的 Region 副本与 TiKV 中完全对应，且会跟随 TiKV 中的 Leader 副本同时进行分裂与合并。
 
-在 Linux AMD64 架构的硬件平台上部署 TiFlash 时，CPU 必须支持 AVX2 指令集。确保命令 `cat /proc/cpuinfo | grep avx2` 有输出。通过使用 AVX2 指令集，TiFlash 的向量化引擎能提供更好的性能。
+在 Linux AMD64 架构的硬件平台上部署 TiFlash 时，CPU 必须支持 AVX2 指令集。确保命令 `cat /proc/cpuinfo | grep avx2` 有输出。而在 Linux ARM64 架构的硬件平台上部署 TiFlash 时，CPU 必须支持 ARMv8 指令集。确保命令 `cat /proc/cpuinfo | grep 'crc32' | grep 'asimd'` 有输出。通过使用 AVX2 和 ARMv8 指令集，TiFlash 的向量化引擎能提供更好的性能。
```
Comment (shichun-0415) [inline tiflash/tiflash-overview.md:20]:
```suggestion
在 Linux AMD64 架构的硬件平台部署 TiFlash 时，CPU 必须支持 AVX2 指令集。确保命令 `cat /proc/cpuinfo | grep avx2` 有输出。而在 Linux ARM64 架构的硬件平台部署 TiFlash 时，CPU 必须支持 ARMv8 指令集。确保命令 `cat /proc/cpuinfo | grep 'crc32' | grep 'asimd'` 有输出。通过使用 AVX2 和 ARMv8 指令集，TiFlash 的向量化引擎能提供更好的性能。
```

File: upgrade-tidb-using-tiup.md [hunk 8]
```diff
@@ -35,7 +35,7 @@ aliases: ['/docs-cn/dev/upgrade-tidb-using-tiup/','/docs-cn/dev/how-to/upgrade/u
     3. 将集群升级至 4.0 版本。
     4. 按本文档说明将集群升级到 6.5.0 版本。
 - 支持 TiDB Binlog，TiCDC，TiFlash 等组件版本的升级。
-- 将 v6.3.0 之前的 TiFlash 升级至 v6.3.0 及之后的版本时，需要特别注意：在 Linux AMD64 架构的硬件平台下部署 TiFlash 时，CPU 必须支持 AVX2 指令集。具体请参考 [6.3.0 版本 Release Notes](/releases/release-6.3.0.md#其他) 中的描述。
+- 将 v6.3.0 之前的 TiFlash 升级至 v6.3.0 及之后的版本时，需要特别注意：在 Linux AMD64 架构的硬件平台下部署 TiFlash 时，CPU 必须支持 AVX2 指令集。而在 Linux ARM64 架构的硬件平台上部署 TiFlash 时，CPU 必须支持 ARMv8 指令集。具体请参考 [6.3.0 版本 Release Notes](/releases/release-6.3.0.md#其他) 中的描述。
```
Comment (shichun-0415) [inline upgrade-tidb-using-tiup.md:38]:
```suggestion
- 将 v6.3.0 之前的 TiFlash 升级至 v6.3.0 及之后的版本时，需要特别注意：在 Linux AMD64 架构的硬件平台部署 TiFlash 时，CPU 必须支持 AVX2 指令集。而在 Linux ARM64 架构的硬件平台部署 TiFlash 时，CPU 必须支持 ARMv8 指令集。具体请参考 [6.3.0 版本 Release Notes](/releases/release-6.3.0.md#其他) 中的描述。
```

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Review by shichun-0415 — APPROVED** [review]

**Pull request merged**